tests/compute/test_kernel.py

```diff
@@ -69,7 +74,7 @@ def _test(red):
             g.update_all(fn.copy_src(src='u', out='m'),
                          builtin[red](msg='m', out='r1'))
             r1 = g.ndata['r1']
-            F.backward(r1.sum())
+            F.backward(F.reduce_sum(r1))
```

Comment:
I suppose `reduce_sum` means taking the sum of all elements?

Reply:
Yes

Reply:
OK, then I would refactor my code for pooling layers following the same naming style.